File: tacas2020-benchmarks/booster/condm.c
#define N

int main()
{
	int i;
	int a[N];

	for (i = 0; i < N; i++)
	{
		a[i] = 0;
	}

	for (i = 0; i < N; i++)
	{
		if (N%2 == 0) {
			a[i] = a[i] + 2;
		} else {
			a[i] = a[i] + 1;
		}
	}

	for (i = 0; i < N; i++)
	{
		assert(a[i] % 2 == N % 2);
	}
	return 1;
}

File: tacas2020-benchmarks/vaphor/condm.java
class vapexp{
	static int i,N;
	static int a[];

static void main()
{
	N=Support.random();
	i=0;
	while(i<N)
	{
		a[i] = 0;
		i = i + 1;
	}

	i=0;
	while(i<N)
	{
		if (N%2 == 0)
		{
			a[i] = a[i] + 2;
		}
		else
		{
			a[i] = a[i] + 1;
		}
		i = i + 1;
	}

	i=0;
	while(i<N)
	{
		assert(a[i] % 2 == N % 2);
		i = i + 1;
	}
}
}
